h2o-algos/src/test/java/hex/coxph/CoxPHTest.java

```diff
@@ -264,12 +276,18 @@ public void testCoxPHSingleNodeMode() {
       parms._response_column = "event";
       parms._ignored_columns = new String[]{"id", "year", "surgery", "transplant"};
       parms._ties = CoxPHModel.CoxPHParameters.CoxPHTies.efron;
+      
+      // Concordance computation can't support single node so it's disabled for this test
+      System.setProperty("sys.ai.h2o.debug.skipScoring", Boolean.TRUE.toString());
```

Comment:
This needs to be in try-finally block or it will not be unset when assertion fails

Reply:
It's inside try-finally if I look correctly.

Reply:
my fault! sorry, you are right!